src/utils/idb.js

```diff
@@ -0,0 +1,21 @@
+function getIDB() {
+    /* global indexedDB,webkitIndexedDB,mozIndexedDB,OIndexedDB,msIndexedDB */
+    if (typeof indexedDB !== 'undefined') {
+        return indexedDB;
+    }
+    if (typeof webkitIndexedDB !== 'undefined') {
+        return webkitIndexedDB;
+    }
+    if (typeof mozIndexedDB !== 'undefined') {
+        return mozIndexedDB;
+    }
+    if (typeof OIndexedDB !== 'undefined') {
+        return OIndexedDB;
+    }
+    if (typeof msIndexedDB !== 'undefined') {
+        return msIndexedDB;
+    }
+}
```

Comment:
Another change I made in a lot of places is to avoid checking the `global` object (e.g. `global.indexedDB`) and instead just check `typeof indexedDB !== 'undefined'`. I've found that this is less faily for weird environment like NodeWebkit where you can't be sure what the global object is even called. It should also make the code more portable if you do want to switch between Webpack, Browserify, SystemJS, etc.